README.md

```diff
@@ -8,25 +8,54 @@ The website for the PySlackers slack community - a place for python learners, te
 
 Prereqs:
 
-* `docker`
-* `python3.6+`
+* docker
+* python3.6+
+
+The following external services are required
+
+* PostgreSQL 9.6+
+* Redis
+
+How you install these services is up to you, but the easiest way is by using
+Docker and Docker compose.
+
+1. Install Docker and Docker Compose by following the instructions on the [Docker website](https://docs.docker.com/compose/install/)
+
+2. Run Docker Compose in daemon mode `-d`:
+
+    `docker-compose up -d`
+
+3. If not running the containers in daemon mode open a new terminal window.
+
+4. Create a virtualenv and install dependencies.
+
+    `python3 -m venv .venv && source .venv/bin/activate && pip install -e .`
+
+5. Activate the virtualenv.
+
+    `source .venv/bin/activate`
+
+6. Run Django migrations.
+
+    `PY_ENV=development ./manage.py migrate`
+
+7. Create a superuser.
+    
+    `./manage.py createsuperuser`
+
+8. Run the Django devserver.
+
+    `./manage.py devserver`
```

Comment:
Same issue with py_env. I was going to default it for making Dev easier, but the cost would be prod being potentially unsafe (I'd rather make Dev setup one minor thing extra)